### PR TITLE
Cpdp 1632 fix1

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html [hunk 1]
```diff
@@ -66,21 +66,30 @@ <h2 class="heading-medium" id="subsection-title">
...
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet1", "#overview"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet2", "#howTo"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet3", "#beforeYouStart"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet4", "#ifAgent"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet5", "#ifTrust"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet6", "#capasitorOrPersonalRepresentative"))</li>
+        </ol>
+        <h2 class="heading-medium" id="overview">@messages("landingPage.intro.h2")</h2>
+    } else {
+        @Html("")
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:81]: do we not want the contents bit to still be there if capacitors/personal reps are disabled?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html [hunk 2]
```diff
@@ -144,27 +166,31 @@ <h2 class="heading-medium">@messages("landingPage.reportingDisposals.h2")</h2>
...
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p5"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p6"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p7"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p8"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.example.title"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p9"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p10"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p11"))</p>
+    } else {
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p2", appConfig.contactHmrc))</p>
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:194]: did you disable capacitors/personal reps and compare with the existing live version to check that it's still the same?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html (hunk 1 above)
Comment (modivilioglu) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:81]: For the individuals, there was no contents bit initially. So we are behaving like the prev.

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:81]: ah ok, I forgot that I'm looking at a diff with the new content on the left already

**Comment by AndyHWChung** [comment]
LGTM

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html (hunk 2 above)
Comment (modivilioglu) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:194]: Yes. Done it for both.

**Pull request merged**